_query-dsl/specialized/script.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: default
+title: Script query
+parent: Specialized queries
+nav_order: 58
+---
+
+# Script query
+
+Use the `script` query to filter documents based on a custom condition written in the Painless scripting language. This query returns documents for which the script evaluates to `true`, enabling advanced filtering logic that can't be expressed using standard queries.
+
+The `script` query is computationally expensive and should be used sparingly. Only use it when necessary and ensure `search.allow_expensive_queries` is enabled (default is `true`).
+
+## Parameters
+
+| Parameter       | Required/Optional | Description                                           |
+| --------------- | ----------------- | ----------------------------------------------------- |
+| `script.source` | Required          | The script code that evaluates to `true` or `false`.  |
+| `script.params` | Optional          | User-defined parameters referenced inside the script. |
+
+## Create a sample index
```

Comment:
Let's call this section "Example" and move it to before the "Parameters" section to mimic other query pages. And instead of headings for steps, add descriptive intro sentences to the requests.